src/lib/lessons.ts

```diff
@@ -1,30 +1,7 @@
 import {LessonResource} from 'types'
 import {getGraphQLClient} from '../utils/configured-graphql-client'
-import config from './config'
 import getAccessTokenFromCookie from '../utils/get-access-token-from-cookie'
 
-export async function loadLessons(): Promise<LessonResource[]> {
```

Comment:
The component that used this function was removed back in August 2020.
It hasn't been used since.
https://github.com/eggheadio/egghead-next/commit/ff184c09e66a66f4b7a9ed0f7aae649962408dc5

Plus we are trying to remove data loading dependence on egghead-rails,
especially for lesson metadata.